spec/controllers/ems_common_controller_spec.rb

```diff
@@ -169,7 +169,24 @@
 end
 
 describe EmsContainerController do
+  before :each do
+    allow(controller).to receive(:get_hostname_from_routes).and_return("nshneor-centos7-1.eng.lab.tlv.redhat.com")
```

Comment:
This feels like an expectation that should live in the test, not a stub for every test in the whole controller.